static/src/js/routes/Project/Project.js

```diff
@@ -66,24 +65,10 @@ export class Project extends Component {
     const { search } = location
     const { edscHost } = this
 
-    // If there are no params in the URL, show the saved projects page
+    // If there are no project params in the URL, re-route to the saved Projects page
     if (search === '') {
```

Comment:
I'm a bit confused I'll pull this down and test it myself but, if the conditional here `if (search === '')` is failing to properly work i.e. its `false` when it should be true, which is why we are getting routed to the other component wouldn't this still have the same issue even if  it is on a difference page?

Reply:
So the issue I seem to be seeing is that something like `http://localhost:8080/saved-projects?projectId=3519105921`is always going to strip out the projectId but, also that `http://localhost:8080/projects?projectId=3519105921` is redirecting me (fill in any projectId) back into the `saved-projects` page so I can't go back to my previous projects